.github/workflows/ci-integration-test-environment-variables.yml

```diff
@@ -0,0 +1,40 @@
+name: CI - integration test environment-variables input
```

Comment:
As discussed on slack, please add another integration test which leverages the new `runs-on` support so that we can be sure that this also works on windows